<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthWnd</class>
 <widget class="QDialog" name="AuthWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">                  QPushButton
                  {
                  background: #890709;
                  text-align: center;
                  color: #FFFFFF;
                  }
                  QPushButton::hover{
                  background: #890709;
                  border: 1px solid #7B59FF;
                  }

                  QPushButton::pressed{
                  background: #890709;
                  }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineLogin_">
           <property name="placeholderText">
            <string>Логин...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="linePassword_">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>Пароль...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLogin_">
           <property name="font">
            <font/>
           </property>
           <property name="text">
            <string>Вход</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRegister_">
           <property name="font">
            <font/>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Регистрация</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineRegName_">
           <property name="echoMode">
            <enum>QLineEdit::Normal</enum>
           </property>
           <property name="placeholderText">
            <string>Логин...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineRegPasswd_">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>Пароль...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCompleteReg_">
           <property name="text">
            <string>Зарегистрироваться</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
